pkg/build/ldconfig.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2022 Chainguard, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package build
+
+import (
+	"debug/elf"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+
+	apkofs "chainguard.dev/apko/pkg/fs"
+)
+
+// original alpine ldconfig available at https://git.alpinelinux.org/aports/tree/main/musl/ldconfig
```

Comment:
For Wolfi this needs to run proper `/sbin/ldconfig` because of glibc `/etc/ld.so.cache`.